android-components/components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksUseCases.kt

```diff
@@ -183,10 +183,9 @@ class AppLinksUseCases(
                     // no default app found but Android resolver shows there are multiple applications
                     // that can open this app link
                     ANDROID_RESOLVER_PACKAGE_NAME, null -> {
-                        findActivities(appIntent).filter {
-                            it.filter != null &&
-                                !(it.filter.countDataPaths() == 0 && it.filter.countDataAuthorities() == 0)
-                        }.getOrNull(0)
+                        findActivities(appIntent).firstOrNull {
+                            it.filter != null
+                        }
                     }
```

Comment:
Is it possible to write tests to verify this new behaviour?  Or the existing ones already covers it?

Reply:
I'll check to see if there's a good test for this.  Thanks

Reply:
@Amejia481 I couldn't find anything good test to add here.  Existing tests already covered what we need.

Reply:
Thanks for the clarification :)